### ISSUE TITLE
Remove Nativefier apps

### ISSUE THREAD

**Issue by theofficialgman** [Botspot/pi-apps#1791]
### Cool ideas?

Maintaining nativefier apps requires extra burden for little benefit.

I personally see zero benefit to using nativefier to packaging webpages into their own electron app. All that does it increase the bloat on a users install (more space taken up by now multiple redundant chromium versions).

there are legitimate uses for electron where other libraries are also used to build the application, but taking a webpage and just throwing it into a separate electron instance isn't one of them.

If you want to keep these apps that use nativefier in pi-apps, my suggestion would be to make install scripts that depend on chromium to be installed, and create .desktop files that launch a unique instance of chromium with tabs disabled and its own folder for settings/passwords/etc (like an electron app would have). you can do this like so: 
```
chromium-browser %U --user-data-dir=$HOME/.config/webapps/WhatsApp --app=https://web.whatsapp.com
```

**Comment by Itai-Nelken** [comment]
We did do yhis for whatsapp a long time ago when natovefier was broken on arm.
Personally I agree, chromium webapps are much faster than nativefier apps.

**Comment by theofficialgman** [comment]
correct me if I'm missing anything but I the only nativefier app left in pi-apps is whatsapp.

I thought there was at least one more but I can't find one.

**Comment by ryanfortner** [comment]
> I thought there was at least one more but I can't find one.

Snapdrop is also a Nativefier app.

**Comment by ryanfortner** [comment]
I agree with you on Electron apps' unnecessarily high resource usage and frequent update requirements. However, I don't really think there's much of a point to having the script simply add a Chromium desktop shortcut. After all, the user could just go to snapdrop.net or whatsapp.com on their own.

I'm just saying I don't think simple chromium desktop shortcuts should be considered apps.

**Comment by theofficialgman** [comment]
> After all, the user could just go to snapdrop.net or whatsapp.com on their own.

right, ofc, but I figured this was better than removing them. 

> I personally think Snapdrop and Whatsapp should either be left as they are, or removed entirely.

I do not think outdated apps that never get updated should stay in pi-apps, especially electron based which are full of bugs that could have security issues for the user.

> Also, Nativefier apps have added features like an optional tray icon.

did either of these apps use this? I didn't notice it but I could have missed it.

**Comment by ryanfortner** [comment]
> > Also, Nativefier apps have added features like an optional tray icon.
> 
> did either of these apps use this? I didn't notice it but I could have missed it.

Yes, I know WhatsApp definitely uses it but I'm not sure about Snapdrop.

**Comment by Botspot** [comment]
IIRC, the advantage to making Snapdrop a Nativefier app was that @Itai-Nelken managed to package the website into the app for first launch. Consequently, if two RPis have snapdrop installed and are connected to the local network but don't have access to the entire Internet, Snapdrop would still work.
This was a bit of an edge case but I think it makes Snapdrop more similar to Apple's Airdrop, which also doesn't require an internet connection. (AFAIK)

**Comment by Botspot** [comment]
> Nativefier simply caches the website it loads, so it can be loaded with no internet connection.

In addition to that, I just tested the proposed Snapdrop Chromium webapp and the Nativefier one - the nativefier one finished loading faster than the Chromium could.

**Comment by theofficialgman** [comment]
I'm fine either way. I just don't want this: "should either be left as they are". electron updates are important for security reasons. also old versions get blacklisted by sites once they user agent is too old.

**Comment by ryanfortner** [comment]
I am requesting that Snapdrop be left as-is. If and when Electron needs updating, I will repackage it.

**Comment by Botspot** [comment]
What timing. I just tried to use snapdrop.net in the web browser and the servers are down. But the Nativefier still works!

**Comment by cycool29** [comment]
> I do not think outdated apps that never get updated should stay in pi-apps, especially electron based which are full of bugs that could have security issues for the user.

WhatsApp debs hosted on my [repo](https://github.com/cycool29/whatsapp-for-linux) are automatically rebuilt every month.

**Comment by theofficialgman** [comment]
ok I think for both of these apps the usecase for nativefier is clear. lets keep in mind everything said here for future nativefier apps that might be wanted in pi-apps to make sure they actually add significant features (like offline usage, notifications, etc)
